cases/management/commands/enrich_ciaa_related_entities.py

```
@@ -0,0 +1,477 @@
+                    entity.nes_id = nes_id
+                    entity.save(update_fields=["nes_id"])
+
+                if created:
+                    self.stats["entities_created"] += 1
+
+                relationship_type_enum = (
+                    RelationshipType.LOCATION
+                    if rel_type == "location"
+                    else RelationshipType.RELATED
+                )
+                rel, rel_created = CaseEntityRelationship.objects.get_or_create(
```

Ruff (0.15.15)

[warning] Unpacked variable `rel` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)